h2o-core/src/main/java/water/util/MathUtils.java

```diff
@@ -66,10 +66,13 @@ static public double computeWeightedQuantile(Vec weight, Vec values, double alph
     private final double[] _mean;
     private final double[] _m2;
     double[] _wsums;
+    double[] _wsumsq; // store sum of squares of original element
```

Comment:
I would avoid modifying this class if there is another way to do it. Adding anything here will make things just more confusing :)

Reply:
Done implementing the apache class for std and mean